src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs

```diff
@@ -353,9 +376,17 @@ public ActionResult Row1Panel1(AccountDashboardViewModel model)
                 {
                     viewModel.ViewName = "SignAgreement";
                 }
-                else if (!model.HasReservations && model.ApprenticeshipEmployerType == Common.Domain.Types.ApprenticeshipEmployerType.NonLevy)
+                else if (model.ApprenticeshipEmployerType == Common.Domain.Types.ApprenticeshipEmployerType.NonLevy)
                 {
-                    viewModel.ViewName = "CheckFunding";
+                    if (model.ReservationsCount == 1 && model.ConfirmedReservationsCount == 1 && !model.ApprenticeshipAdded)
```

Comment:
we should be checking for an ConfirmedReservationCount = 0. "Confirmed", means that is has been used on an apprenticeship. "Pending" means that is created but as yet unassigned